api/helpers/statusMessagesHelper.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    200: 'Ok',
+    201: 'Created',
+    202: 'Accepted',
+    204: 'Not Content',
+    301: 'Moved Permanently',
+    302: 'Found',
+    303: 'See Other',
+    304: 'Not Modified',
+    307: 'Temporary Redirect',
+    400: 'Bad Request',
+    401: 'Unauthorized',
+    403: 'Forbidden',
+    404: 'Not Found',
+    405: 'Method Not Allowed',
+    406: 'Not Acceptable',
+    412: 'Precondition Failed',
+    415: 'Unsupported Media Type',
+    500: 'Internal Server Error',
+    501: 'Not Implemented',
+}
```

Comment:
Remove unused status codes, if we will need some status code we just will add him.

Reply:
This status is a base instance for HTTP protocol.

Reply:
It just an example that you could to add some custom codes to your response.